_includes/nav.html

```diff
@@ -7,64 +7,67 @@
       <div id="user-nav">
         <a href="/" class="waves-effect"><em><span style="font-variant:small-caps">emergency</span> INDEX</em></a>
       </div>
-
-      <div class="input-field">
-        <input type="text" id="autocomplete-input" class="autocomplete">
-        <label for="autocomplete-input" class='search-label'> Search</label>
-      </div>
     </li>
 
     <ul class="collapsible-nav" data-collapsible="accordion">
       <li class="collapsible-nav-item">
         <div class="collapsible-header sticky-nav sticky-nav-2 waves-effect {% if page.layout == 'abouts' %}active{% endif %}">About <i class="material-icons indicator">keyboard_arrow_down</i></div>
         <div class="collapsible-body">
           <ul>
-            <li class="waves-effect {% if page.permalink contains 'submit-your-performance' %}active{% endif %}">
-              <a href="/submit-your-performance/" class="about">Submit Your Performance</a>
-            </li>
-            <li class="waves-effect {% if page.permalink contains 'purchase' %}active{% endif %}">
-              <a href="/purchase/" class="about">Purchase a Book</a>
-            </li>
             <li class="waves-effect {% if page.permalink contains 'masthead' %}active{% endif %}">
               <a href="/about/masthead/" class="about">Masthead</a>
             </li>
-            <li class="waves-effect {% if page.permalink contains 'partner-organizations' %}active{% endif %}">
-              <a href="/about/partner-organizations/" class="about">Partner Organizations</a>
-            </li>
             <li class="waves-effect {% if page.permalink contains 'project-description' %}active{% endif %}">
               <a href="/about/project-description/" class="about">Project Description</a>
             </li>
             <li class="waves-effect {% if page.permalink contains 'press' %}active{% endif %}">
               <a href="/about/press/" class="about">Press</a>
             </li>
+            <li class="waves-effect {% if page.permalink contains 'purchase' %}active{% endif %}">
+              <a href="/purchase/" class="about">Purchase/Subscribe</a>
+            </li>
           </ul>
         </div>
       </li>
       <li class="collapsible-nav-item">
-        <div class="collapsible-header sticky-nav sticky-nav-2 waves-effect">Volumes <i class="material-icons indicator">keyboard_arrow_down</i></div>
+        <div class="collapsible-header sticky-nav sticky-nav-2 waves-effect {% if page.layout == 'archives' || 'volume' || 'project' %}active{% endif %}">Archive <i class="material-icons indicator">keyboard_arrow_down</i></div>
```

Comment:
you need to compare the `page.layout` var for everything between the `||`z here: 

```ruby
if page.layout == 'archives' ||  page.layout == 'volume' ||  page.layout == 'project'
```